CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+ ## Not yet released
+
+### Features and enhancements
+
+* [ide] Improve patch board design that shows bound values (#1684)
+* [ide] Make sure nodes and comments are always aligned to slots grid (#1695)
+* [ide] Show a list of available inputs/outputs in QuickHelp when C++ or tabtest editors are open (#1682)
+* [ide] Make tabline scrollable if too many tabs are open (#1694)
+* [ide] Close tabs on middle click (#1686)
+* [nodes] Add descriptions to all `xod/patch-nodes` and auto-generated terminals for custom types (#1685)
+
+### New nodes
+
+* [nodes] Add `tweak-pulse` node (#1701)
+* [nodes] Add `xod/math/min` and `xod/math/max` (#1700)
```

Comment:
…and `integrate-dt`

The `[nodes]` marker makes a little sense in this section

Reply:
Done!